catalog/core/catalog-core-api/src/main/java/ddf/catalog/data/Metacard.java

```diff
@@ -219,6 +219,13 @@
     public static final String DERIVED = "metacard.associations.derived";
 
     /**
+     * {@link Attribute} name for external associations.
+     *
+     * @since DDF-2.10.0
+     */
+    public static final String EXTERNAL = "metacard.associations.external";
```

Comment:
Doesn't this only belong in : https://github.com/codice/ddf/blob/770409a9fbed37172c21aeeb77f0bf6ff7e3913f/catalog/core/catalog-core-api/src/main/java/ddf/catalog/data/types/Associations.java